### ISSUE TITLE
Ender S1 Pro - no leading zeroes on minutes in print progress monitor

### ISSUE THREAD

**Issue by sadgit** [CrealityOfficial/Ender-3S1#53]
version 2.0.8 26F4

When displaying progress "One Hour and Five Minutes" is displayed as 1:5

It should be displayed as 1:05

Thanks

**Comment by wizardkitty** [comment]
With how the S1 Pro firmware is this would have to be in the screen firmware.

**Comment by ThomasToka** [comment]
The bug here indeed is in the screen firmware here on sites 1,9,10 and 12. "Display invalid zeros" is deactivated for this field.

I have already fixed this in my development branch an will deliver this fix with my v002 release.

![IMG_5270](https://github.com/CrealityOfficial/Ender-3S1/assets/117008525/91445637-df2a-464b-876e-65559be0f66d)

**Comment by ThomasToka** [comment]
this is how it looks on this site in the editor:

![grafik](https://github.com/CrealityOfficial/Ender-3S1/assets/117008525/18c7b7f4-d5d4-4d55-b2ad-56f879b993d3)

**Comment by jklap** [comment]
I don't see any screen files in your repo? Where are you keeping these types of changes @ThomasToka ?

**Comment by ThomasToka** [comment]
v002 is not released yet. you can download v001 with its screen with many enhancements in my v001 release https://github.com/ThomasToka/MarlinFirmware/releases/tag/MARLIN-E3S1PROFORK-BYTT-v001

**Comment by jklap** [comment]
I'm not asking about the compiled firmware, I'm asking about the src files for the screen work you've done :) Does that not exist in your Marlin repo?

**Comment by ThomasToka** [comment]
yes sure. naturally i release all source files after my final release.. https://github.com/ThomasToka/Marlin/tree/MARLIN-E3S1PROFORK-BYTT/ScreenFirmware

if you want to compile the v001 state you need to take this https://github.com/ThomasToka/Marlin/commits/MARLIN-E3S1PROFORK-BYTT-RELEASE-CODE . other branches are mid development as i continue developing  ;)

**Comment by jklap** [comment]
Ah, it's in zip files. Thanks!

**Comment by ThomasToka** [comment]
today i released v002 with the mentioned fix, ubl meshviewer and other fixes. I created a new branch for the release @wizardkitty.

Naturally all in the release. Including the screen sources.
